packages/jest-environment-puppeteer/README.md

```diff
@@ -113,19 +113,22 @@ This example uses an already running instance of Chrome by passing the active we
 
 ```js
 // jest-puppeteer.config.js
-const wsEndpoint = fs.readFileSync(endpointPath, 'utf8')
+const PromiseForConfig = PromiseForWsEndpoint
```

Comment:
Your example is not enought explicit. Can you please make it more explicit? I mean `PromiseForWsEndpoint` should be defined. Also no first uppercase for promises please, they are not classes.

Reply:
Thanks!